surveys/templates/survey_submitted_detail.html

```diff
@@ -2,11 +2,18 @@
 {% load static %}
 
 {% block extracss %}
-  <link rel="stylesheet" href="{% static 'css/datatables.min.css' %}">
+  <link rel="stylesheet" href="{% static 'css/dataTables.bootstrap4.min.css' %}">
+
+  <link rel="stylesheet" href="{% static 'css/buttons.bootstrap4.min.css' %}">
 {% endblock %}
 
 {% block extrajs %}
-  <script type="text/javascript" src="{% static 'js/datatables.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'js/jquery.dataTables.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'js/dataTables.bootstrap4.min.js' %}"></script>
+
+  <script type="text/javascript" src="{% static 'js/dataTables.buttons.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'js/buttons.bootstrap4.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'js/buttons.html5.min.js' %}"></script>
```

Comment:
Could you say a little bit about what these file changes are accomplishing?

Reply:
Following the DataTables [documentation](https://datatables.net/extensions/buttons/examples/initialisation/export.html):

> Buttons provides button types that will automatically determine if HTML5 or Flash should be used based on the browser's functionality and it is strongly recommended that you use these button types over the specific HTML5 or Flash button types. These are: copy, csv, excel, pdf.

I'd forgotten the flash js file and I've added it here!